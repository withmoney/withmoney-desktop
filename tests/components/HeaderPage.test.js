import renderer from 'react-test-renderer';
import HeaderPage from '../../src/components/HeaderPage';

describe('HeaderPage', () => {
  let Component;

  beforeAll(() => {
    Component = global.setup(HeaderPage);
  });

  it('should render a component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
