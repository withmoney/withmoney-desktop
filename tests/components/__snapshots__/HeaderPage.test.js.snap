// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderPage should render a component correctly 1`] = `
<div
  className="header"
>
  <div
    className="header__container"
  >
    <div
      className="header__row"
    >
      <div
        className="header__title"
      >
        withmoney
      </div>
      <div
        className="menu-user"
      >
        <div
          className="menu-user__name"
        >
          David Costa
        </div>
        <img
          alt="avatar"
          className="menu-user__avatar"
          src="/static/img/avatar.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
